packages/types/src/interfaces/augment-api-tx.ts

```diff
@@ -2061,89 +1579,73 @@ declare module '@polkadot/api/types/submittable' {
        * - DbWrites: `Reasons`, `Tips`
        * # </weight>
        **/
-      tipNew: AugmentedSubmittable<
-        (
-          reason: Bytes | string | Uint8Array,
-          who: AccountId | string | Uint8Array,
-          tipValue: Compact<BalanceOf> | AnyNumber | Uint8Array
-        ) => SubmittableExtrinsic<ApiType>
-      >;
+      tipNew: AugmentedSubmittable<(reason: Bytes | string | Uint8Array, who: AccountId | string | Uint8Array, tipValue: Compact<BalanceOf> | AnyNumber | Uint8Array) => SubmittableExtrinsic<ApiType>>;
       /**
        * Unassign curator from a bounty.
-       *
+       * 
        * This function can only be called by the `RejectOrigin` a signed origin.
-       *
+       * 
        * If this function is called by the `RejectOrigin`, we assume that the curator is malicious
        * or inactive. As a result, we will slash the curator when possible.
-       *
+       * 
        * If the origin is the curator, we take this as a sign they are unable to do their job and
        * they willingly give up. We could slash them, but for now we allow them to recover their
        * deposit and exit without issue. (We may want to change this if it is abused.)
-       *
+       * 
        * Finally, the origin can be anyone if and only if the curator is "inactive". This allows
        * anyone in the community to call out that a curator is not doing their due diligence, and
        * we should pick a new curator. In this case the curator should also be slashed.
-       *
+       * 
        * # <weight>
        * - O(1).
        * - Limited storage reads.
        * - One DB change.
        * # </weight>
        **/
-      unassignCurator: AugmentedSubmittable<
-        (bountyId: Compact<ProposalIndex> | AnyNumber | Uint8Array) => SubmittableExtrinsic<ApiType>
-      >;
+      unassignCurator: AugmentedSubmittable<(bountyId: Compact<ProposalIndex> | AnyNumber | Uint8Array) => SubmittableExtrinsic<ApiType>>;
     };
+    // @ts-ignore
```

Comment:
need to investigate the reason why auto-generate adds utility, when it already exist.. @ts-ignored for now

Reply:
Similarly, the metadata information that comes from CENNZnet node for utility pallet is different from latest substrate..
it has additional function named - batchAll.. and some more weight related information..
<img width="1780" alt="Screen Shot 2020-11-30 at 2 14 12 PM" src="https://user-images.githubusercontent.com/29415595/100559204-80e94380-3316-11eb-8fb7-1a33ec914588.png">

Reply:
ah I see, do you know if these definitions will merge or overwrite each other?

Reply:
I think it overwrites. cause when I look at api.tx.batch object or even in UI.. I don't see batchAll